string-manipulation-2/src/ransom-case.js

```diff
@@ -0,0 +1,12 @@
+/* exported ransomCase */
+function ransomCase(string) {
+  if (typeof string !== 'string') {
+    return 'Invalid';
+  } else {
+    var lower = string.toLowerCase().split('');
```

Comment:
Tip: You don't have to split into array if you are iterating through each character for string since `string[0]` will be first character, `string[1]` will be second char and so on